webpack.config.js

```diff
@@ -0,0 +1,18 @@
+const path = require('path');
```

Comment:
Please, remove this file, because it doesn't allow to build an app properly

Reply:
Removed!